tests/pytorch/nightly/python-ops.libsonnet

```diff
@@ -39,9 +39,24 @@ local utils = import 'templates/utils.libsonnet';
   local v3_8 = {
     accelerator: tpus.v3_8,
   },
+  local tpuVm = common.PyTorchTpuVmMixin {
+    tpuSettings+: {
+      tpuVmExports+: |||
+        export XLA_USE_BF16=$(XLA_USE_BF16)
+      |||,
+      tpuVmExtraSetup: |||
+        pip install tensorboardX google-cloud-storage
```

Comment:
Are these installs all necessary? As far as I know, the python ops test does not require tensorboard or fairseq

Reply:
Thanks, Will, updated